README.md

```diff
@@ -1,8 +1,78 @@
-# aws-elasticsearch-example [Work in progress]
+# aws-elasticsearch-example
 
 ## About
 
-This repository has a series of examples for migrating your cloud.gov applications to use the new Elasticsearch service using [AWS Elastisearch](https://aws.amazon.com/elasticsearch-service/).
+This repository is an example for using the Elasticsearch service in your cloud.gov applications
+from using the `elasticsearch56` service running on the `medium-ha` and `medium` plans to the
+`elasticsearch-aws` service running on the `aws-elasticsearch-prod` and `aws-elasticsearch-dev` plans powered by
+[AWS Elastisearch](https://aws.amazon.com/elasticsearch-service/). The new AWS Elasticsearch services provides customers with access the Elasticsearch v7.4; upgrading the current Elasticsearch offering from v5.4.
```

Comment:
The last part might imply that CG will be upgrading the old ES service to the new version. Thoughts?

Reply:
It does.

Reply:
From a customer point of view, I think that CG will be doing the upgrade and/or migration for ES instane and I only need to make sure my app works.